src/site/cookies/index.njk

```diff
@@ -0,0 +1,101 @@
+---
+title: Cookies on the MOD.UK Design System
```

Comment:
Believe the title is meant to just be Cookies in the footer so we may need another property (and use `title` as a fallback perhaps)

Reply:
Saw the comment on Jira that the page title should be Cookies as well so this is not irrelevant now